pkg/java/pom/parse_test.go

```diff
@@ -32,6 +32,7 @@ func TestPom_Parse(t *testing.T) {
 				{
 					Name:    "com.example:happy",
 					Version: "1.0.0",
+					License: "Apache 2.0",
```

Comment:
Looks like all licenses are `Apache 2.0` other than multiply licenses. Can we change one of them to another license so that we can make sure the test identifies licenses correctly? This prevents bugs where values are not updated properly, and all values are the same by mistake.

Reply:
done